### PR TITLE
Flysystem Adapters Fix

### PR DESCRIPTION
**Pull request by mrgodhani**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
The other filesystems need fixing too.

**Comment by mrgodhani** [comment]
Eh All the adapters are out of the main repository.

**Comment by GrahamCampbell** [comment]
No they're not. Rackspace isn't. Sftp isn't. Dropbox isn't.

**Comment by mrgodhani** [comment]
http://flysystem.thephpleague.com/upgrade-to-1.0.0/

**Comment by GrahamCampbell** [comment]
I thought you meant out as in came from, rather than removed from. Yeh, I know what's been going on. I've contributed to most of them, and spent part of the last two days getting my laravel flysystem wrapper ready. :)

**Comment by mrgodhani** [comment]
So I guess dependency for flysystem needs to be updated over here.

**Comment by GrahamCampbell** [comment]
#55. This whole issue is caused by using g a version constraint with no upper bound.

**Comment by mrgodhani** [comment]
ye and also we still need to add adapters. So I guess added suggestions to install required adapters. And changed classes.

File: composer.json [hunk 1]
```diff
@@ -38,7 +38,11 @@
         "dropbox/dropbox-sdk": "Enables Dropbox storage",
```
Comment (GrahamCampbell) [inline composer.json]: Please remove this entry,

File: composer.json [hunk 2]
```diff
@@ -38,7 +38,11 @@
...
         "phpseclib/phpseclib": "Enables SFTP storage",
```
Comment (GrahamCampbell) [inline composer.json]: Please remove this entry,

File: composer.json [hunk 3]
```diff
@@ -38,7 +38,11 @@
...
         "sabre/dav": "Enables WebDav storage",
```
Comment (GrahamCampbell) [inline composer.json]: Please remove this entry,

File: composer.json [hunk 4]
```diff
@@ -38,7 +38,11 @@
...
-        "rackspace/php-opencloud": "Enables Rackspace Cloud storage"
+        "rackspace/php-opencloud": "Enables Rackspace Cloud storage",
```
Comment (GrahamCampbell) [inline composer.json]: Please remove this entry,

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [inline composer.json]: And the aws one. GitHub won't let me comment on it.

**Comment by mrgodhani** [comment]
Done.

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Comment by mrgodhani** [comment]
Done

**Comment by GrahamCampbell** [comment]
The commit name is wrong, lol. Otherwise, :+1: from me. :)

**Comment by mrgodhani** [comment]
Lol I renamed commit name.

**Comment by GrahamCampbell** [comment]
Oh noes, now we've got a merge commit. Did you forget to force push?

**Comment by GrahamCampbell** [comment]
Also, "Flysystem" is spelt like that, without an uppercase "S".

**Comment by mrgodhani** [comment]
I did git push origin master -f I think it is suppose to be git push -f aargh now I have to fix this

**Comment by mrgodhani** [comment]
ok Fixed.

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by ShawnMcCool** [comment]
Thanks to both of you.

**Comment by GrahamCampbell** [comment]
@ShawnMcCool This pull actually breaks the test. It seems travis wasn't even running them. I'm working on a fix...

**Comment by ShawnMcCool** [comment]
@GrahamCampbell If you ever need a job reference, let me know.

**Comment by GrahamCampbell** [comment]
lol

**Comment by ShawnMcCool** [comment]
@GrahamCampbell just a heads up, i went ahead and fixed it all

**Comment by GrahamCampbell** [comment]
@ShawnMcCool Great. :)

**Pull request merged**